derp/derp_server.go

```diff
@@ -1024,9 +1037,7 @@ func (s *Server) recordDrop(packetBytes []byte, srcKey, dstKey key.NodePublic, r
 		msg := fmt.Sprintf("drop (%s) %s -> %s", srcKey.ShortString(), reason, dstKey.ShortString())
 		s.limitedLogf(msg)
 	}
-	if s.debug {
-		s.logf("dropping packet reason=%s dst=%s disco=%v", reason, dstKey, disco.LooksLikeDiscoWrapper(packetBytes))
-	}
+	s.debugLogf("dropping packet reason=%s dst=%s disco=%v", reason, dstKey, disco.LooksLikeDiscoWrapper(packetBytes))
```

Comment:
This will call `LooksLikeDiscoWrapper` on every dropped packet even when debug logging is not enabled - perhaps it's worth keeping the result of a [previous call](https://github.com/tailscale/tailscale/blob/f0ee03dfafda9c5a8804b25f20ea881090bd4ab5/derp/derp_server.go#L1014) in a variable and use it here?

Reply:
Good catch! Thank you, fixed as you describe.